src/main/steps/common/contact-us.njk

```diff
@@ -0,0 +1,27 @@
+<details class="govuk-details" data-module="govuk-details">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      {{ contactUsForHelp }}
+    </span>
+  </summary>
+  <div class="govuk-details__text">
+    <ul class="govuk-list contact-us">
+      <li>
+        <h4 class="govuk-heading-s">{{ webChat }}</h4>
+        <p class="govuk-body">{{ webChatDetails }}</p>
```

Comment:
Looks like `webChatDetails` will always contain "All our web chat agents are busy helping other people". 

Is there something that dynamically update this when there is someone available to help?

Reply:
there was no dynamic content/google docs or info on the ticket about it. i know there is another ticket for webchat and i assumed that would be related to that ticket.